services/web-app/components/page-tabs/page-tabs.js

```diff
@@ -6,18 +6,28 @@
  */
 import { Column, Grid, Tab, TabList, Tabs } from '@carbon/react'
 import clsx from 'clsx'
+import { useRouter } from 'next/router'
 import PropTypes from 'prop-types'
+import slugify from 'slugify'
 
 import styles from './page-tabs.module.scss'
 
 const PageTabs = ({ className, tabs = [] }) => {
+  const router = useRouter()
+
+  const currTabSlug = router.asPath.split('/').pop()
+
+  const currTabIndex = tabs.findIndex(
+    (tab) => slugify(tab.name, { strict: true, lower: true }) === currTabSlug
+  )
+
   return (
     <Grid className={clsx(styles.container, className)} narrow>
       <Column sm={4} md={8} lg={12}>
-        <Tabs>
-          <TabList aria-label="List of tabs" className={styles['tab-list']}>
+        <Tabs defaultSelectedIndex={currTabIndex > 0 ? currTabIndex : 0}>
+          <TabList aria-label="List of tabs" className={styles['tab-list']} selected>
             {tabs.map((tab, i) => (
-              <Tab href={tab.path} key={i}>
+              <Tab as="a" href={tab.path} key={i}>
                 {tab.name}
               </Tab>
```

Comment:
```suggestion
              <Tab as="span" key={i}>
                <Link href={tab.path}>
                  <a href={tab.path} className={styles['tab-link']}>
                    {tab.name}
                  </a>
                </Link>
              </Tab>
```

I think we want to use the next/link component here so the page doesn't do a full refresh when you switch tabs. I added as span because nesting a link inside a button seemed odd. (This will need some styling fixes)

```
.tab-link {
  color: theme.$text-primary;
  text-decoration: none;
}
```